frontend/src/app/organization/organization.model.ts

```diff
@@ -1,6 +1,9 @@
-interface Organization {
+export class Organization {
```

Comment:
I think there is nothing wrong with the interface (exported, of course) here. Class would make sense if you created objects yourself. You will not, as you won't create id, slug or url.